### DISCUSSION TITLE
Live reloading Remix deps within a monorepo (turborepo)

### DISCUSSION THREAD

**Discussion by koriner** [remix-run/remix#1930] [Q&A]
This is a question around using turborepo alonside Remix, but probably applies to other monorepo options as well.

I've made an app that uses Remix and targets Vercel, based on the example here: https://github.com/remix-run/remix/tree/main/examples/turborepo-vercel

Another similar post is found here: https://dev.to/bronifty/remix-in-turbo-nx-workspaces-monorepo-comparison-3b1f

The main point with this Vercel target is that the actual packages within the repo need to have a build step added. Each package seems to need it's own little build process so you get a compiled module to use.

That works fine, and everything does work, but not the hot/live reloading in the Remix app, of changes that happen in the package code. For example if I make a change in `packages/ui/index.tsx` and save it, the build runs, but Remix won't detect the changes until I stop and restart the turborepo script to run the dev server. As an aside, a NextJS app within the same `apps/` dir will reload as expected.

It's been pointed out that Remix doesn't allow customisation of what directories to watch for changes, it just defaults to the actual Remix project folder.

Is there a way around this? Am I doing something silly?

Comments welcome!

**Comment by nareshbhatia** [comment]
I have a similar issue filed on the Turborepo side: https://github.com/vercel/turborepo/issues/1035
